tests/testsuite/profiles.rs

```diff
@@ -468,3 +468,34 @@ fn thin_lto_works() {
         )
         .run();
 }
+
+#[cargo_test]
+fn strip_works() {
+    let p = project()
```

Comment:
I think just above this you'll need a `return;` if this isn't the nightly channel (there's some other tests that show examples of doing this)